api/swagger.yaml

```diff
@@ -1,3 +1,14 @@
+# A Swagger 2.0 (a.k.a. OpenAPI) definition of the Engine API.
+#
+# This is used for generating API documentation and the types used by the
+# client/server. See api/README.md for more information.
+#
+# Some style notes:
+# - This file is used by ReDoc, which allows GitHub Flavored Markdown in
+#   descriptions.
+# - There is no maximum line length, for ease of editing and pretty diffs.
```

Comment:
I could remove the comment, if you'd like, then maybe people would start truncating to 80 chars. ;)

Converting the file now to 80 chars would be quite a large job. I don't think it can be easily automated because of all the Markdown, etc.